doc/changelog/02-specification-language/13188-instance-gen.rst

```diff
@@ -0,0 +1,6 @@
+- **Removed:** The type given to :cmd:`Instance` is not automatically
+  generalized over free variables any more. Replace ``Instance :
+  type`` by ``Instance : \`{type}`` to reproduce the old behaviour, or
+  enable compatibility option :opt:`Instance Generalized Output`.
```

Comment:
```suggestion
- **Removed:** The type given to :cmd:`Instance` is no longer automatically
  generalized over free variables. Use :n:`Instance : \`{@type}` instead of :n:`Instance :
  @type` to get the old behaviour, or
  enable the compatibility option :opt:`Instance Generalized Output`.
```

@Zimmi48 Is there any guideline for preferring British or American spelling in the doc?  FWIW, there are 11 uses of "behaviour" and 143 uses of "behavior" in the rsts.  Consistency is good.  (Also FWIW, I would not be very good at correcting to British spelling!)

Reply:
Ah, I edited this after checking it with Sphinx, but you got the earlier version.  It needs the `%{` escapes, also splitting `:n:` across a line end makes it a forced newline in the HTML, which doesn't look too good:

![image](https://user-images.githubusercontent.com/1253341/97921311-9b2c0380-1d0f-11eb-846d-fcea8f086faf.png)